exla/Makefile

```
@@ -9,7 +9,7 @@ XLA_INCLUDE_PATH = $(XLA_EXTENSION_DIR)/include
 
 # Cache configuration
 EXLA_CACHE_SO = cache/libexla.so
```

Shouldn't we version this one as well? Just in case?

```suggestion
EXLA_CACHE_SO = cache/$(EXLA_VERSION)/libexla.so
```

---

Shouldn't hurt :)
I'd tried this during development but it had failed. Seems to be working locally now.